close #4957 by linking to slice() from splice() pg

hirehamir opened the pull request:
This commit tries to close https://github.com/mdn/content/issues/4957.

<!-- Please provide the following information to help us review this PR: -->

> What was wrong/why is this fix needed? (quick summary only)

I think it would be helpful to readers if we linked to [`Array.prototype.slice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) from [`Array.prototype.splice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)'s page since a reader might look for one while on the page of the other.
> MDN URL of the main page changed

[`Array.prototype.splice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)
> Issue number (if there is an associated issue)

#4957 
> Anything else that could help us review it

hirehamir committed:
close #4957 by linking to slice() from splice() pg

This commit tries to close https://github.com/mdn/content/issues/4957.

wbamberg commented on files/en-us/web/javascript/reference/global_objects/array/splice/index.html:
```diff
@@ -16,7 +16,7 @@
 <p>The <strong><code>splice()</code></strong> method changes the contents of an array by
   removing or replacing existing elements and/or adding new elements <a
-    href="https://en.wikipedia.org/wiki/In-place_algorithm">in place</a>.</p>
+    href="https://en.wikipedia.org/wiki/In-place_algorithm">in place</a>. If you would like to access part of an array without modifying it, please see {{jsxref("Array.prototype.slice()", "slice()")}} instead.</p>
```
```suggestion
    href="https://en.wikipedia.org/wiki/In-place_algorithm">in place</a>. To access part of an array without modifying it, see {{jsxref("Array.prototype.slice()", "slice()")}}.</p>
```

wbamberg reviewed (changes requested):
Thanks @hamirmahal ! I agree that this is a nice cross-link, but I had some editorial comments (as a rule, I like to keep docs concise and direct).

hirehamir committed:
make recommendation to see slice page more concise

This commit addresses https://github.com/mdn/content/pull/4976#pullrequestreview-659361545.

Co-authored-by: wbamberg <will@bootbonnet.ca>

hamirmahal commented:
You're welcome, @wbamberg! I like your suggestion and committed it. Thanks for the improvement!

wbamberg reviewed (approved):
👍 looks good, thank you @hamirmahal !

hamirmahal commented:
You're welcome, @wbamberg! Happy I could help.
